common/src/main/java/com/box/l10n/mojito/github/GithubClient.java

```diff
@@ -57,10 +59,12 @@ public GithubClient(String appId, String key, String owner, long tokenTTL) {
     }
     this.owner = owner;
     this.tokenTTL = tokenTTL;
+    this.endpoint =
+        endpoint.endsWith("/") ? endpoint.substring(0, endpoint.length() - 1) : endpoint;
   }
 
   public GithubClient(String appId, String key, String owner) {
-    this(appId, key, owner, 60000L);
+    this(appId, key, owner, 60000L, "https://api.github.com");
```

Comment:
can we grab that from their constant "GITHUB_URL"?

Reply:
No, it's not a public constant so can't be accessed outside of the `org.kohsuke.github` package